variables.tf

```diff
@@ -21,6 +21,12 @@ variable "allowed_methods" {
   description = "Controls which HTTP methods CloudFront processes and forwards"
 }
 
+variable "application_logo" {
+  type        = string
+  default     = null
+  description = "Local path to the application logo image"
```

Comment:
Define `Local` in this context? Should this be `Relative` instead?

Reply:
Relative might cover it better. Commit incoming